components/experimental/src/transliterate/transliterator/mod.rs

```diff
@@ -382,13 +396,15 @@ impl Transliterator {
         })
     }
 
+    #[allow(clippy::too_many_arguments)]
     fn load_rbt<PT, PN, PC, F>(
         marker_attributes: &DataMarkerAttributes,
         lookup: Option<&F>,
         transliterator_provider: &PT,
         normalizer_provider: &PN,
         casemap_provider: &PC,
         allow_internal: bool,
+        fallbacker: impl Fn() -> Result<LocaleFallbacker, DataError>,
```

Comment:
Thought: You can't use the fallbacker built into databake because you're falling back within data marker attributes? Maybe worth adding a comment explaining this, since it is unusual ICU4X style to explicitly take a fallbacker in a constructor.